src/error.js

```diff
@@ -0,0 +1,4 @@
+/* @flow */
+export const ERROR_CODE = {
```

Comment:
nit: can you add a new line here?
```suggestion

export const ERROR_CODE = {
```

Reply:
@gregjopa Done. Do I need to squash the commit?